specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/bastionHost.json

```diff
@@ -0,0 +1,440 @@
+                },
+                {  
+                   "$ref":"./network.json#/parameters/ApiVersionParameter"
+                },
+                {  
+                   "$ref":"./network.json#/parameters/SubscriptionIdParameter"
+                }
+             ],
+             "responses":{  
+                "200":{  
+                   "description":"Request successful. The operation returns an BastionHost resource.",
+                   "schema":{  
+                      "$ref":"#/definitions/BastionHost"
+                   }
+                }
+             },
+             "x-ms-examples":{  
+                "Get Bastion Host":{  
+                   "$ref":"./examples/BastionHostGet.json"
+                }
+             }
+          },
+          "put":{  
+             "tags":[  
+                "BastionHosts"
+             ],
+             "operationId":"BastionHosts_CreateOrUpdate",
+             "description":"Creates or updates the specified Bastion Host.",
+             "parameters":[  
+                {  
+                   "name":"resourceGroupName",
+                   "in":"path",
+                   "required":true,
+                   "type":"string",
+                   "description":"The name of the resource group."
+                },
+                {  
+                   "name":"bastionHostName",
+                   "in":"path",
+                   "required":true,
+                   "type":"string",
+                   "description":"The name of the Bastion Host."
+                },
+                {  
+                   "name":"parameters",
+                   "in":"body",
+                   "required":true,
+                   "schema":{  
+                      "$ref":"#/definitions/BastionHost"
+                   },
+                   "description":"Parameters supplied to the create or update Bastion Host operation."
+                },
+                {  
+                   "$ref":"./network.json#/parameters/ApiVersionParameter"
+                },
+                {  
+                   "$ref":"./network.json#/parameters/SubscriptionIdParameter"
+                }
+             ],
+             "responses":{  
+                "201":{  
+                   "description":"Create successful. The operation returns the resulting BastionHost resource.",
+                   "schema":{  
+                      "$ref":"#/definitions/BastionHost"
+                   }
+                },
+                "200":{  
+                   "description":"Update successful. The operation returns the resulting BastionHost resource.",
+                   "schema":{  
+                      "$ref":"#/definitions/BastionHost"
+                   }
+                }
+             },
+             "x-ms-examples":{  
+                "Create Bastion Host":{  
+                   "$ref":"./examples/BastionHostPut.json"
+                }
+             },
+             "x-ms-long-running-operation":true
```

Comment:
Please add the code below for all operations containing `"x-ms-long-running-operation": true`
```
        "x-ms-long-running-operation-options": {
          "final-state-via": "location"
        },
```

Reply:
`"final-state-via": "azure-async-operation"` for `PUT` requests

Reply:
done